diff --git a/dist/playwright.js b/dist/playwright.js
index 461d4f54c60a8ef11894fa94f824f7e1da4789fe..bd59d78394b8b303abffdefb6404ca52ace8cb4b 100644
--- a/dist/playwright.js
+++ b/dist/playwright.js
@@ -35,8 +35,7 @@ var IframeTransform = class {
 		const testerUi = iframe.parentElement;
 		if (!testerUi) throw new Error(`Cannot find Tester element. This is a bug in vitest-browser-commands. Please, open a new issue with reproduction.`);
 		const scaleAttribute = testerUi.getAttribute("data-scale");
-		const scale = Number(scaleAttribute);
-		if (Number.isNaN(scale)) throw new TypeError(`Cannot parse scale value from Tester element (${scaleAttribute}). This is a bug in vitest-browser-commands. Please, open a new issue with reproduction.`);
+		const scale = Number(scaleAttribute) || 1;
 		const rect = iframe.getBoundingClientRect();
 		this.iframeX = rect.x;
 		this.iframeY = rect.y;
